lib/index.js

```diff
@@ -85,12 +89,22 @@ function start (config) {
       setImmediate(() => { outputStream.write(`${JSON.stringify(data)}\n`, 'utf8', cb) })
     } else {
       // otherwise send out stdout and do not block the loop.
-      setImmediate(() => { process.stdout.write(`${JSON.stringify(data)}\n`, 'utf8', cb) })
+      if (output && output.notjson) {
```

Comment:
* Please rename this option to `isJson`
* Add documentation to `README.md` to showcase this as an available option.